systemd/systemd.go

```diff
@@ -285,6 +299,37 @@ func (c *Collector) collect(ch chan<- prometheus.Metric) error {
 	return nil
 }
 
+func (c *Collector) collectBootStageTimestamps(conn *dbus.Conn, ch chan<- prometheus.Metric) error {
+	stages := []string{"Finish", "Firmware", "Loader", "Kernel", "InitRD",
```

Comment:
Hmm, there's no way to get the stage list dynamically is there?

Reply:
Not without embedding some of the C code - I doubt the list will change much going forward.